aten/src/ATen/native/cpu/UpSampleKernel.cpp

```diff
@@ -1310,13 +1314,32 @@ struct HelperInterpCubic : public HelperInterpBase {
             scale,
             interp_size,
             &HelperInterpCubic::aa_filter<scalar_t>,
-            /*antialias=*/true,
-            /*align_corners_delta*/0.0);
+            /*antialias=*/antialias,
+            /*align_corners_delta*/align_corners_delta);
       }
     );
     return indices_weights;
   }
 
+  static inline std::tuple<std::vector<Tensor>, int, unsigned int> compute_indices_int16_weights_aa(
```

Comment:
It will be called once we can implement int32 buffer support inside `cpu_upsample_generic_aa`. It is also confusing to see a generic templated code with hard-coded bilinear mode for bytes.